packages/server/manage_container/server/libc_to_manage_container/process_data.go

```diff
@@ -0,0 +1,94 @@
+package l2mserver
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"fmt"
+	"math/big"
+
+	m2db "github.com/acompany-develop/QuickMPC/packages/server/manage_container/client/manage_to_db"
+	utils "github.com/acompany-develop/QuickMPC/packages/server/manage_container/utils"
+)
+
+type processer struct {
+	m2dbclient m2db.M2DbClient
+}
+
+func (p processer) addShareDataFrame(baseDataID string, addDataID string) (string, error) {
+	// pieceサイズの取得(とデータの存在確認)
+	basePieceSize, errBasePiece := p.m2dbclient.GetSharePieceSize(baseDataID)
+	if errBasePiece != nil {
+		return "", errBasePiece
+	}
+	addPieceSize, errAddPiece := p.m2dbclient.GetSharePieceSize(addDataID)
+	if errAddPiece != nil {
+		return "", errAddPiece
+	}
+
+	// 新しいデータIDの生成
+	s := fmt.Sprintf("%s+%s", baseDataID, addDataID)
+	r := sha256.Sum256([]byte(s))
+	dataID := hex.EncodeToString(r[:])
```

Comment:
When duplicates are found, return them as is.

Reply:
@KotaTakahashi9320 
I fixed it!
[6475808](https://github.com/acompany-develop/QuickMPC/pull/242/commits/64758086777db9a5baf56d64050af3620e3e2f5d)